lib/saxerator/builder/hash_builder.rb

```diff
@@ -60,12 +60,13 @@ def add_to_hash_element(hash, name, element)
 
       def block_variable
         return to_s if @text
-        return to_hash if @children.count > 0 || (@attributes.count > 0 && @config.put_attributes_in_hash?)
+        return to_hash if @children.count > 0 ||
+                          (@attributes.count > 0 && @config.put_attributes_in_hash?)
```

Comment:
I strongly dislike whitespace to align things into columns. I'd rather extract methods or variables to keep this short enough to not have to wrap. Any cop that promotes that style should be disabled.